enterprise_access/settings/base.py

```diff
@@ -140,6 +141,14 @@ def root(*path_fragments):
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
 }
 
+SPECTACULAR_SETTINGS = {
+    'TITLE': 'Enterprise Access API',
+    'DESCRIPTION': 'API for controlling request-based access to enterprise subsidized enrollments.',
+    'VERSION': '1.0.0',
+    'ENUM_ADD_EXPLICIT_BLANK_NULL_CHOICE': False,
```

Comment:
We do not need this setting, this setting is only useful for `drf_spectacular.hooks.postprocess_schema_enums` hook and we are not using that here.